### PR TITLE
Login/register UI improvements + minor changes

### PR DESCRIPTION
**Pull request by MortenMunk**
## Description

Minor improvements/polishing of login and register + UI/UX improvements

## Changes

- adjusted indentation in EducadoModel.js
- Replaced Educado Logo as PNG to SVG, and made 'fill' a parameter for the color of the SVG in EducadoLogo.js
-  used new educado logo component in LogoBackButton.js and added styling to align backbutton with logo
- added login functionality to RegisterForm.js, so that the user is logged in and navigated to home page, after registering
- added more comments to RegisterForm.js
- Imported KeyboardAwareScrollView for Login.js and Register.js to have the page scrollable when keyboard is up
- added tests to Register.js
- updated snapshots in tests related to above changes
- Added z-index styling to logoBackButton, so back button is still pressable
- added width/height props to EducadoLogo
- consolidated imports in register
- added a test for scrolling on register.js and login.js
- updated tests to match new test guidelines

## Related Issues

None

## Checklist

- [X] Code has been tested locally and passes all relevant tests.
- [X] Documentation has been updated to reflect the changes, if applicable.
- [X] Code follows the established coding style and guidelines of the project.
- [X] All new and existing tests related to the changes have passed.
- [X] Any necessary dependencies or new packages have been properly documented.
- [X] Pull request title and description are clear and descriptive.
- [X] Reviewers have been assigned to the pull request.
- [X] Any potential security implications have been considered and addressed.
- [X] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)

None relevant

## Notes for Reviewers

When using the new EducadoLogo component, it is possible to change its color to any of the custom tailwind colors we have defined, using this syntax:
```
<EducadoLogo fill='fill-primary' />
```
Whereas fx. black could also be another tailwind defined color, then it would be fill='fill-black'. This is specified in the doxygen comment of the component.

> **Notice!**\
> - Responsive scaling for SVG components is not perfect yet... fx. it may not be noticeable at width='50%' on login, whereas width='20%' is noticeable. I suspect this is because of the viewbox of SVGs, but this will need another PBI to fix, as responsiveness is not a criteria for this PBI.
> - The old Educado PNG should eventually be removed, once teams adapt this new SVG.

### CODE REVIEW

File: eml/components/general/EducadoModal.js
Comment (Andazu) [inline eml/components/general/EducadoModal.js:1]: Padding values are pixels, and not percentages, reconsider?

File: eml/screens/register/Register.js [hunk 1]
```diff
@@ -1,42 +1,68 @@
 import React from "react";
-import { View, TouchableWithoutFeedback, Keyboard } from "react-native";
+import { View, TouchableWithoutFeedback, Keyboard, ScrollView } from "react-native";
 import { useNavigation } from "@react-navigation/native";
 import RegisterForm from "../../components/login/RegisterForm";
 import { SafeAreaView } from "react-native-safe-area-context";
 import LogoBackButton from "../../components/login/LogoBackButton";
 import Text from "../../components/general/Text";
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
+import { useEffect } from "react";
```
Comment (Andazu) [inline eml/screens/register/Register.js:9]: Can this be added to the import statement in the first line?

**Review by Andazu — COMMENTED** [review]
Looks good, biggest comment is to remember responsiveness, many values are pixels and not percentages

**Review by JackSMunn — APPROVED** [review]
I approve of this message. I agree with @Andazu regarding % instead of PX for responsiveness

**Review by TheMOHD3886 — APPROVED** [review]
sinds kode 👍

**Review by Andazu — APPROVED** [review]

**Pull request merged**